### PR TITLE
Skip update without builds when creating compose

### LINKED ISSUE #3471: Update without build can be created
It appears that an update without build can be created (https://bodhi.fedoraproject.org/updates/FEDORA-2019-e4e61bb84b). 
We need to make sure that the validators are not letting an update without a build being created.

In that case the update made the compose failed.

### PR DESCRIPTION
**Pull request by Zlopez**
Fixes #3471

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,4 @@ nosetests.xml
 virtenv
 .vscode
 test_results
+.projectile
```
Comment (abompard) [inline .gitignore]: Could you explain what this is please?

Comment (Zlopez) [reply .gitignore]: This is Emacs projectile module. I started to use it recently.

Comment (pypingou) [reply .gitignore]: You can add this to ~/.gitignore and this will then be ignored in all your git repositories

Comment (Zlopez) [reply .gitignore]: @pypingou This will be probably better solution, I hope it is working on the Silverblue now. Last time I tried it, it didn't worked.

Comment (Zlopez) [reply .gitignore]: `~/.gitignore` is still ignored, but I removed it from `.gitignore`

**Review by pypingou — APPROVED** [review]
Looks fine to me

**Pull request merged**